### PR TITLE
test(e2e): added team member related scenarios

### PR DESCRIPTION
**Pull request by LaxSoma**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { ProjectTeamPage } from '../../page_objects/projectTeamPage';
+import { projectInformation } from '../../support/utils/createProjectInformation';
+
+const createCasePage = new CreateCasePage();
+const projectTeamPage = new ProjectTeamPage();
+const applicationsHomePage = new ApplicationsHomePage();
+const searchResultsPage = new SearchResultsPage();
+const { applications: applicationUsers } = users;
+
+describe('Create case ', () => {
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js:17]: better name this: `Case Team` or something like that

Comment (LaxSoma) [reply apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js:17]: Sure

File: apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+	it('Verify team member is added', () => {
+		cy.login(applicationUsers.caseAdmin);
+		cy.visit('/');
+		const caseRef = Cypress.env('currentCreatedCase');
+		applicationsHomePage.searchFor(caseRef);
+		searchResultsPage.clickTopSearchResult();
+		searchResultsPage.clickLinkByText('Project team');
+		searchResultsPage.clickButtonByText('Add team member');
+		projectTeamPage.searchTeamMemberByEmail('caseofficeradmin.test@planninginspectorate.gov.uk');
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js:46]: when/if `caseofficeradmin.test@planninginspectorate.gov.uk` is decommissioned this test will not return results

Comment (LaxSoma) [reply apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js:46]: caseofficer.test@planninginspectorate.gov.uk is decommissioned not caseofficeradmin.test@planninginspectorate.gov.uk

Comment (LaxSoma) [reply apps/e2e/cypress/e2e/back-office-applications/addTeam.spec.js:46]: sure

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**